src/components/BounceDetailsContainer/Modals/ConfirmationModal/index.jsx

```diff
@@ -88,20 +93,25 @@ const ConfirmationModal = ({
   handleSaveConfirmation,
   onChangeRule,
   isUpdateError,
-}) => (
-  <CenterModal
-    {...WriteSelectors.saveConfirmationModal}
-    open
-    renderBody={(
-      <ConfirmModalBody
-        updatedRule={updatedRule}
-        handleModalClose={handleModalClose}
-        handleSaveConfirmation={handleSaveConfirmation}
-        onChangeRule={onChangeRule}
-        isUpdateError={isUpdateError}
-      />
+}) => {
+  const { comment } = updatedRule;
+  const isCommitEmpty = comment === undefined || comment.length <= 0;
```

Comment:
Does it already show an error upon first showing the commit input? We should have it be showing no inline error upon first showing empty commit input and only when you backspace from having input it should show the inline error. The save button will be disabled only if it's empty/has error

Reply:
fixed this ^